### DISCUSSION TITLE
Is there any open source repo for Text Toolbar / Editor mentioned in the document?

### DISCUSSION THREAD

**Discussion by Smilefounder** [carbon-design-system/carbon#8618] [Help]
Hi IBM team,

Kindly advise if there is an opensource repo for the Text Editor mentioned here:
https://www.carbondesignsystem.com/patterns/text-toolbar-pattern/

Best regards,
Huy Nguyen

**Comment by jnm2377** [comment]
Hi @Smilefounder, it looks like based on this issue: https://github.com/carbon-design-system/carbon/issues/2447, we accepted the proposal for the component design/pattern but a coded asset was never contributed. Are you interested in contributing it?
